packages/app-form-builder/src/components/Form/FormRender.tsx

```diff
@@ -73,8 +89,12 @@ const FormRender: React.FC<FbFormRenderComponentProps> = props => {
         return fields.find(field => field.fieldId === id) || null;
     };
 
-    const getFields = (): FormRenderFbFormModelField[][] => {
-        const fieldLayout = cloneDeep(layout);
+    // We need to have "stepIndex" prop in order to get corresponding fields for the current step.
+    const getFields = (stepIndex: number): FormRenderFbFormModelField[][] => {
```

Comment:
Let's have `stepIndex = 0` here, just so if a user knows he's working with single-step forms, he can just call `getFields()` and that's it.